src/scripts.js

```diff
@@ -43,8 +43,8 @@ const addClass = (elements, rule) => {
     elements.forEach(item => item.classList.add(rule));
 }
 
-const displayCurrentRecipes = (currentRecipes) => {
-  let display = currentRecipes;
+const displayCurrentRecipes = () => {
```

Comment:
Great decision not needing to pass anything in here! We can just reassign the global cookbook instance to be whatever the user's favorite recipes are!